source/index.txt

```diff
@@ -66,9 +68,29 @@ core operations. This section contains information on how to:
 API Documentation
 -----------------
 
-The `API documentation <{+api+}/apidocs/mongodb-driver-sync/>`__
-has detailed technical information about classes, methods, and
-configuration objects within the MongoDB Java driver.
+The MongoDB Java driver API documentation is split into several sections.
+Each section has detailed technical information about classes, methods, and
+configuration objects within the MongoDB Java driver:
```

Comment:
I think it's important to communicate that the API documentation is organized by package or library (contained in JARs). While it is true that it is split into sections, each section covers exactly one JAR and not parts of another JAR. Similarly, I would use package or library in the header of the table instead of "Section".